.github/PROCESS.md

```diff
@@ -73,13 +73,14 @@ For FerretDB Inc. engineers and long-term contributors, besides the guides in [C
    (because [words like "refs" do not link PR to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)),
    but don't forget to reopen the issue once PR is merged,
    but the issue as a whole is not done.
-5. Please create a draft pull request as soon as you start working on an issue; that is needed for our process tooling.
+5. (This point is for FerretDB Inc. engineers only.)
```

Comment:
If I understand we mention that, as PROCESS.md applies also to long-term contributors

Reply:
Yes, and also because we ask for the opposite in CONTRIBUTING.md